ddtrace/appsec/_handlers.py

```diff
@@ -425,4 +424,4 @@ def listen():
 core.on("flask.patch", _on_flask_patch)
 
 core.on("asgi.request.parse.body", _on_asgi_request_parse_body, "await_receive_and_body")
-core.on("grpc.response_message", _on_grpc_response, "response")
+core.on("grpc.response_message", _on_grpc_response)
```

Comment:
Why we need to make this change? Does the name of the callback matter?

Reply:
The name (third parameter) is the key used to store the return value of the listener when calling `dispatch_with_results`, since we are not calling with `dispatch_with_results` anymore, the name is not necessary.

No harm in leaving it, but I figured it might help to signify that it is not supposed to return a value.